src/operations/revoke_sponsorship.js

```diff
@@ -63,7 +63,7 @@ export function revokeTrustlineSponsorship(opts = {}) {
   if (!StrKey.isValidEd25519PublicKey(opts.account)) {
     throw new Error('account is invalid');
   }
-  if (!(opts.asset instanceof Asset)) {
+  if (!(opts.asset instanceof TrustLineAsset)) {
```

Comment:
Oh hm I guess we *do* do a type-check on assets in other places... Okay, up to you to keep it above then!

Reply:
That's where I got the inspiration to add this validation 😅 . I've decided to let the validations there but I'm open to removing them if you feel strongly towards it.

Reply:
Yep, makes sense! I'm totally fine with keeping it.